app-next/components/account/ElderPageComponent.jsx

```diff
@@ -0,0 +1,256 @@
+"use client";
+import { useAddressStore } from "@/store/useAddressStore";
+import React, { useEffect, useState } from "react";
+const initialAddress = {
+  name: "",
+  address: "",
+  postalcode: "",
+  city: "",
+  country: "",
+  phone: "",
+  isDefault: false,
+};
+export default function ElderPageComponent({ user }) {
+  const { createAddress, address, error, isLoading, fetchAddress } =
+    useAddressStore();
+  const [helpRequests, setHelpRequests] = useState([]);
+  const [showAddress, setShhowAddress] = useState(false);
+  const [formData, setFormdata] = useState(initialAddress);
+  const addresses = Array.isArray(address) ? address : [];
+  const handleAddAddress = async (event) => {
+    event.preventDefault();
+    try {
+      const result = await createAddress(formData);
+      if (result) {
+        fetchAddress();
+        setFormdata(initialAddress);
+        setShhowAddress(false);
+        alert("AddressCreated Successfully");
+      }
+    } catch (error) {
+      console.log(error);
+      alert(error);
+    }
+  };
+  useEffect(() => {
+    fetchAddress();
+  }, [fetchAddress]);
+  const handleEditAddress = async (addressId) => {
+    console.log(addressId);
+  };
+  const handleDeleteAddress = async (addressId) => {
+    console.log(addressId);
+  };
```

Comment:
Do we have separate tasks to implement this logic? If yes -- good, if no -- we need either create them or delete unused logic